pilot/pkg/features/pilot.go

```diff
@@ -242,6 +242,9 @@ var (
 
 	PreferDestinationRulesTLSForExternalServices = env.Register("PREFER_DESTINATIONRULE_TLS_FOR_EXTERNAL_SERVICES", true,
 		"If true, external services will prefer the TLS settings from DestinationRules over the metadata TLS settings.").Get()
+
```

Comment:
Just curious: Instead of making so many changes on the release branch, can we use the same upstream PR and modify the OSSM Operator helm values to set this env variable to "true" by default?